src/workerd/api/queue.c++

```diff
@@ -433,18 +435,24 @@ QueueEvent::QueueEvent(jsg::Lock& js, Params params, IoPtr<QueueEventResult> res
   messages = messagesBuilder.finish();
 }
 
-void QueueEvent::retryAll() {
+void QueueEvent::retryAll(jsg::Optional<QueueRetryOptions> options) {
   if (result->ackAll) {
     IoContext::current().logWarning(
         "Received a call to retryAll() after ackAll() was already called. "
         "Calling retryAll() after calling ackAll() has no effect.");
     return;
   }
-  result->retryAll = true;
```

Comment:
Similar to above, shouldn't we continue setting the old field until we know that no code is still around that consumes from it?

Reply:
Queues code currently merges both fields.